### PR TITLE
Update logging code sample.

### PR DESCRIPTION
**Pull request by cesarbs**
cc @danroth27 @BrennanConroy

### CODE REVIEW

File: aspnet/fundamentals/logging/sample/src/TodoApi/project.json [hunk 1]
```diff
@@ -6,17 +6,16 @@
...
-    "System.Diagnostics.TextWriterTraceListener": "4.0.0-beta-23516"
+    "Microsoft.AspNetCore.Mvc": "1.0.0",
+    "Microsoft.AspNetCore.StaticFiles": "1.0.0",
+    "Microsoft.AspNetCore.Server.Kestrel": "1.0.0",
+    "Microsoft.Extensions.Logging": "1.0.0",
+    "Microsoft.Extensions.Logging.Console": "1.0.0",
+    "Microsoft.Extensions.Logging.TraceSource": "1.0.0",
+    "Microsoft.AspNetCore.Server.IISIntegration": "1.0.0",
+    "Microsoft.Extensions.Logging.Filter": "1.0.0",
+    "System.Diagnostics.TextWriterTraceListener": "4.0.0-rc2-24027"
```
Comment (BrennanConroy) [inline aspnet/fundamentals/logging/sample/src/TodoApi/project.json]: What is the 1.0.0 release version 4.0.0? Shouldn't we use that here?

Comment (cesarbs) [inline aspnet/fundamentals/logging/sample/src/TodoApi/project.json]: I don't see a 4.0.0 version: https://www.nuget.org/packages/System.Diagnostics.TextWriterTraceListener/4.0.0-rc2-24027

Will it be available by the time we RTM?

cc @muratg @Eilon @davidfowl

Comment (Eilon) [inline aspnet/fundamentals/logging/sample/src/TodoApi/project.json]: Everything will be RTM at RTM time 😄 At least, certainly everything in CoreFX and ASP.NET. So this doesn't look right.

File: aspnet/fundamentals/logging/sample/src/TodoApi/project.json [hunk 2]
```diff
@@ -6,17 +6,16 @@
...
+    "Microsoft.Extensions.Logging.Console": "1.0.0",
+    "Microsoft.Extensions.Logging.TraceSource": "1.0.0",
+    "Microsoft.AspNetCore.Server.IISIntegration": "1.0.0",
+    "Microsoft.Extensions.Logging.Filter": "1.0.0",
+    "System.Diagnostics.TextWriterTraceListener": "4.0.0"
   },
-
   "tools": {
     "Microsoft.AspNetCore.Server.IISIntegration.Tools": {
       "version": "1.0.0-*",
```
Comment (BrennanConroy) [inline aspnet/fundamentals/logging/sample/src/TodoApi/project.json]: `-*` is bad?

File: aspnet/fundamentals/logging/sample/src/TodoApi/project.json [hunk 3]
```diff
@@ -59,7 +56,6 @@
   "runtimes": {
```
Comment (BrennanConroy) [inline aspnet/fundamentals/logging/sample/src/TodoApi/project.json]: this section shouldn't be needed

**Comment by cesarbs** [comment]
Updated.

**Comment by Rick-Anderson** [comment]
@cesarbs  did you build https://docs.asp.net/en/latest/fundamentals/logging.html to make sure it renders correctly or should I do that?

File: aspnet/fundamentals/logging/sample/src/TodoApi/project.json [hunk 4]
```diff
@@ -1,65 +1,51 @@
...
-    "Microsoft.AspNetCore.Mvc": "1.0.0-rc2-final",
-    "Microsoft.AspNetCore.StaticFiles": "1.0.0-rc2-final",
-    "Microsoft.AspNetCore.Server.Kestrel": "1.0.0-rc2-final",
-    "Microsoft.Extensions.Logging": "1.0.0-rc2-final",
-    "Microsoft.Extensions.Logging.Console": "1.0.0-rc2-final",
-    "Microsoft.Extensions.Logging.TraceSource": "1.0.0-rc2-final",
-    "Microsoft.AspNetCore.Server.IISIntegration": "1.0.0-rc2-final",
-    "Microsoft.Extensions.Logging.Filter": "1.0.0-rc2-final",
-    "System.Diagnostics.TextWriterTraceListener": "4.0.0-beta-23516"
+    "System.Diagnostics.TextWriterTraceListener": "4.0.0",
```
Comment (BrennanConroy) [inline aspnet/fundamentals/logging/sample/src/TodoApi/project.json]: Super nit: I think we sort dependencies alphabetically (including system)

**Comment by BrennanConroy** [comment]
Looks good :+1:

**Comment by cesarbs** [comment]
@Rick-Anderson This is ready for merge.

**Pull request merged**